### DISCUSSION TITLE
Subequations and crossref

### DISCUSSION THREAD

**Discussion by XinyanYang95** [quarto-dev/quarto-cli#5582] [Q&A] [answered]
Are there any ways to create subequations and cross-references such as 

xxx (1.1.a)

xxx (1.1.b)

in Quarto book (able to render both the PDF and HTML files)? Thx!

**Comment by mcanouil** [comment, marked as answer]
I am not aware of an easy way, but see other related threads:
- #4136

And take a look at the following issue which list all cross-reference issues/feature requests including few on equations:
- #4944
